actors/builtin/reward/reward_test.go

```diff
@@ -50,6 +51,40 @@ func TestAwardBlockReward(t *testing.T) {
 	})
 }
 
+func TestWithdrawReward(t *testing.T) {
+	actor := rewardHarness{reward.Actor{}, t}
+	owner := tutil.NewIDAddr(t, 101)
+	m := tutil.NewIDAddr(t, 1000)
+	builder := mock.NewBuilder(context.Background(), builtin.RewardActorAddr).
+		WithCaller(builtin.SystemActorAddr, builtin.SystemActorCodeID).
+		WithActorType(m, builtin.StorageMinerActorCodeID).
+		WithActorType(owner, builtin.AccountActorCodeID)
+
+	t.Run("successfully withdraw reward total", func(t *testing.T) {
+		rt := builder.Build(t)
+		actor.constructAndVerify(rt)
+
+		gasreward := abi.NewTokenAmount(10)
+		rt.SetBalance(abi.NewTokenAmount(100000000))
```

Comment:
This is an artificially low balance that is clamping the total reward payable.
- If you intended this, please document it, and please also remove the non-zero gas reward since it is not payable
- If you didn't, set this value to be above the `BlockRewardTarget`, and expect 100*10^18 + 10 (gas reward) tokens to be withdrawn